tests/functional/pv/space_reclaim/test_rbd_reclaimspace_cronjob.py

```
@@ -231,31 +239,20 @@ def test_reclaim_space_cronjob_with_annotation(
 
         helpers.wait_for_resource_state(pvc_obj, pvc_status)
 
-        log.info("add reclaimspace.csiaddons.openshift.io/schedule label to PVC ")
-        OCP(kind=constants.PVC, namespace=self.namespace).annotate(
-            f"reclaimspace.csiaddons.openshift.io/schedule=@{schedule}", pvc_obj.name
-        )
-
         pvc_to_chron_job_dict = self.wait_for_cronjobs(True, 60)
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)